checker/src/org/checkerframework/checker/lock/LockChecker.java

```diff
@@ -8,6 +12,12 @@
 /**
  * @checker_framework.manual #lock-checker Lock Checker
  */
-@TypeQualifiers({ LockHeld.class, LockPossiblyHeld.class })
+// The Lock Checker uses two distinct type qualifier hierarchies:
+// the @LockPossiblyHeld and the @GuardedByInaccessible hierarchies.
+@TypeQualifiers({ LockPossiblyHeld.class, LockHeld.class,
+                  GuardedByInaccessible.class, GuardedBy.class,
+                  javax.annotation.concurrent.GuardedBy.class,
```

Comment:
But for the GuardedBy hierarchy they are only aliases. If they are also declaration annotations, they don't need to be TypeQualifiers, but the Holding logic will need to treat the javax/jcip annotations as aliases also.